plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestTestingPostgreSqlServer.java

```diff
@@ -56,7 +56,7 @@ public void tearDown()
     {
         closeAll(
                 postgreSqlServer,
-                () -> threadPool.shutdownNow());
+                threadPool::shutdownNow);
```

Comment:
this is ok because `threadPool` is known not to be null

(we could consider doing `threadPool` setup in `setUp` and then lambda would be necessary)